src/PrimaryAuthController.js

```diff
@@ -107,6 +107,12 @@ function (Okta, PrimaryAuthForm, CustomButtons, FooterRegistration, PrimaryAuthM
       'focusout input': function (e) {
         $(e.target.parentElement).removeClass('focused-input');
       },
+
+      /**
+       * @deprecated
+       * This event was originally added for tracking purpose via MixPanel.
+       * After migrating to new tracking system, it's unnecessary now.
```

Comment:
While this was our use case, we don't know how other developers may have used this event.

```suggestion
       * This event was originally added for capturing specific usage metrics and is now obsolete.
```